src/material/button/button-base.ts

```diff
@@ -42,6 +42,9 @@ export const MAT_BUTTON_HOST = {
   // Add a class that applies to all buttons. This makes it easier to target if somebody
   // wants to target all Material buttons.
   '[class.mat-mdc-button-base]': 'true',
+  'mat-button-ripple-uninitialized': 'true',
```

Comment:
```suggestion
  'mat-button-ripple-uninitialized': '',
```
You can leave the right-hand side blank and it will add the attribute with no value, which I think is what you want (using this as a marker attribute)